public/app/features/explore/Logs.tsx

```diff
@@ -205,23 +225,41 @@ export class Logs extends PureComponent<Props, State> {
         </div>
         <div className="logs-panel-options">
           <div className="logs-panel-controls">
-            <Switch label="Time" checked={showTime} onChange={this.onChangeTime} transparent />
-            <Switch label="Unique labels" checked={showLabels} onChange={this.onChangeLabels} transparent />
-            <Switch label="Wrap lines" checked={wrapLogMessage} onChange={this.onChangewrapLogMessage} transparent />
-            <ToggleButtonGroup label="Dedup" transparent={true}>
-              {Object.keys(LogsDedupStrategy).map((dedupType: string, i) => (
-                <ToggleButton
-                  key={i}
-                  value={dedupType}
-                  onChange={this.onChangeDedup}
-                  selected={dedupStrategy === dedupType}
-                  // @ts-ignore
-                  tooltip={LogsDedupDescription[dedupType]}
-                >
-                  {dedupType}
-                </ToggleButton>
-              ))}
-            </ToggleButtonGroup>
+            <div className="logs-panel-controls-main">
+              <Switch label="Time" checked={showTime} onChange={this.onChangeTime} transparent />
+              <Switch label="Unique labels" checked={showLabels} onChange={this.onChangeLabels} transparent />
+              <Switch label="Wrap lines" checked={wrapLogMessage} onChange={this.onChangewrapLogMessage} transparent />
+              <ToggleButtonGroup label="Dedup" transparent={true}>
+                {Object.keys(LogsDedupStrategy).map((dedupType: string, i) => (
+                  <ToggleButton
+                    key={i}
+                    value={dedupType}
+                    onChange={this.onChangeDedup}
+                    selected={dedupStrategy === dedupType}
+                    // @ts-ignore
+                    tooltip={LogsDedupDescription[dedupType]}
+                  >
+                    {dedupType}
+                  </ToggleButton>
+                ))}
+              </ToggleButtonGroup>
+            </div>
+            <button
```

Comment:
We are using "old-style" button to keep the Explore UI consistent. In the future, will be changed to <Button />